src/index.js

```diff
@@ -6,6 +6,28 @@ const needle = require('needle')
 const moment = require('moment-timezone')
 const fs = require('fs')
 const Handlebars = require('handlebars')
+const express = require('express');
+
+const Raven = require('./raven')
+const { uploadToMinio, linkForKey } = require('./minio')
+const config = require('./config')
+const createPdf = require('./utils/pdf');
+
+//serving assets to localhost because puppeteer dont allow access to local files for security reasons
+//read the issue here :- https://github.com/puppeteer/puppeteer/issues/1942
+const PORT = process.env.StaticServerPORT || 3500;
```

Comment:
We should serve the static files by running a server as a different process. Certificate worker isn't responsible for serving static files, it's only responsibility is to generate certificates.

Reply:
We can use [http-server](https://www.npmjs.com/package/http-server) for this

Reply:
should I create a new file server.js which will use http-server and then I will execute this file from index.js ?

Reply:
You can change the Dockerfile to have `start.sh` as an entrypoint and the `start.sh` script will do two things.
1. Run the server to serve static assets
2. Run the certificate worker

Reply:
ok will do it

Reply:
I request you to please assign extra 100 bounty points for this as I have not worked with Dockerfile before and I need to spend some time to figure it out.